client/components/Price.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import faker from 'faker';
+import styled from 'styled-components';
+import StarRatings from 'react-star-ratings';
+
+const Wrapper = styled.div`
+  text-align: left;
+`;
+
+const Currency = styled.span`
+  font-size: 22px;
+  font-weight: 800;
+`;
+
+const Reviews = styled.span`
+  font-size: 12px;
+  font-weight: 600;
+`;
+
+const Rule = styled.section`
+  margin-top: 24px;
+  margin-bottom: 24px;
+  border-bottom: 1px solid #d2d2d2
+`;
+
+const price = () => faker.random.number({
+  min: 50,
+  max: 1500
+}).toLocaleString();
+
+const Price = () => (
```

Comment:
You COULD change the name to inform the user stars are being loaded as well? But I understand price is more important so it'd be okay to keep.

Reply:
Thanks for the input! I didn't see these messages until after I merged, but I've implemented the changes and will push them with my next PR! :)